static_src/test/unit/util/cf_api.spec.js

```diff
@@ -935,17 +935,15 @@ describe('cfApi', function() {
       expect(actual).toMatch(new RegExp('organizations'));
       expect(actual).toMatch(new RegExp('users'));
       actual = spy.getCall(0).args[1];
-      expect(actual).toEqual(userActions.receivedOrgUsers);
-      actual = spy.getCall(0).args[2];
-      expect(actual).toEqual(expected);
+      expect(actual).toEqual(jasmine.any(Function));
     });
   });
 
   describe('fetchOrgUserRoles()', function() {
     it(`should call fetch org user roles with org guid and received org user
         roles action and org guid`, function() {
       var expectedOrgGuid = 'zkjvczcvzwexdvzdfa',
-          spy = sandbox.stub(cfApi, 'fetchMany');
+          spy = sandbox.stub(cfApi, 'fetchAllPages');
```

Comment:
(Capturing answer from convo: 👌 to change it. I'll pop that once my updated dev environment is squared away.)